en/docs/authenticate/regular-webapp-oidc.md

```diff
@@ -1,38 +1,70 @@
-# Enable Authentication to a web application using Authorization code grant.
+# Enable Authentication to OAuth/OpenID Connect Web Application
 
-< Introduction - This document contains an step by step guide on to
-enable authentication for a traditional web application. >
+This page guides you through enabling authentication to an OAuth/OpenID Connect web application. 
 
-!!! Tip 
-    [Try Quick Starts](../../get-started/overview/#mobilenative-application)
+---
 
-## Overview
- < Explain the use case with diagrams - can be links to concepts > 
- 
- < Image explaining the scenario>
+This guide assumes you have your own application. If you wish to try out this flow with a sample application, click the button below. 
 
-## Register Application
+<a class="samplebtn_a" href="../../samples/regular-webapp-oidc-sample" target="_blank" rel="nofollow noopener">Try it with the sample</a>
 
- < Description on web app specific config >
+----
+
+{!authenticate/register-an-sp.md!}
+
+----
 
 {!authenticate/oauth-app-config-basic.md!}
 
-| Field                 | Value         | 
-| --------------------- | ------------- | 
-| Service Provider Name | sample- app  |
-| Description           | This is a traditional web application  | 
-| Call Back Url         | https://somehost/url  | 
+----
+
+## Configure the client application
+
+Make the following requests via your application to connect your application to WSO2 IS. 
+
+1. Obtain the authorization code by sending an authorization request to the authorization endpoint.
+
+    !!! tip
+        If you have made PKCE mandatory, include the `code_challenge` and `code_challenge_method` parameters in the request. You can use [an online tool](https://tonyxu-io.github.io/pkce-generator/) to generate PKCE code challenges.
+
+    ```tab="Request Format"
+    https://<host>/oauth2/authorize?scope=openid&response_type=code
+    &redirect_uri=<redirect_uri>
+    &client_id=<client_id>
+    &code_challenge=<PKCE_code_challenge>
+    &code_challenge_method=<PKCE_code_challenge_method>
+    ```
+
+    ```tab="Sample Request"
+    https://localhost:9443/oauth2/authorize?scope=openid&response_type=code
+    &redirect_uri=https://localhost/callback
+    &client_id=YYVdAL3lLcmrubZ2IkflCAuLwk0a
+    &code_challenge=5vEtIy2T-G65yXHc8g5zcJDQXICBzZMrtERq0zhx7hM
+    &code_challenge_method=S256
+    ```
+
+2. Obtain the access token by sending a token request to the token endpoint using the authorization code recieved in step 1, and the client ID and client secret obtained when configuring the service provider.
+
+    !!! tip
+        If you are using PKCE, include the `code_verifier` parameter in the request.
 
-{!authenticate/configure-client-regular-webapp.md!}
+    ```tab="Request Format"
+    curl -i -X POST -u <Client_ID>:<Client_Secret> -k -d 
```

Comment:
shall we explain what these placeholders are? ex: <Client_ID>, <authorization_code>